### PR TITLE
added slider feature

### PR DESCRIPTION
**Pull request by denysM789**

### CODE REVIEW

File: src/components/CoinOverview/CoinOverview.js [hunk 1]
```diff
@@ -77,9 +81,77 @@ class CoinOverview extends React.Component {
...
+                          $
+                          {Intl.NumberFormat("en-US", {
+                            notation: "compact",
+                            maximumFractionDigits: 1,
+                          }).format(obj.total_supply)}
+                        </p>
+                      </Styled24VMarketCapTextWrapper>
+                      <SliderWrapper
+                        height="8px"
+                        width="100%"
```
Comment (yoloOnTheBattlefield) [inline src/components/CoinOverview/CoinOverview.js:142]: is this necessary? do you use this component in another place where width and height is changed?

File: src/components/Sparkline/Sparkline.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+const Sparkline = () => {
+  const [chartData, setChartData] = useState([]);
+  const [chartLabels, setChartLabels] = useState([]);
+
+  const getBitcoinData = async () => {
+    try {
+      const { data } = await axios(
+        "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=50&page=1&sparkline=true&price_change_percentage=1h%2C24h%2C7d"
+      );
+      //console.log(data);
```
Comment (yoloOnTheBattlefield) [inline src/components/Sparkline/Sparkline.js:14]: no commented out code

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/Sparkline/Sparkline.js (hunk 2 above)
Comment (denysM789) [reply src/components/Sparkline/Sparkline.js:14]: sorry for that one!!

File: src/components/CoinOverview/CoinOverview.js (hunk 1 above)
Comment (denysM789) [reply src/components/CoinOverview/CoinOverview.js:142]: changed. thank you!!

**Pull request merged**